v19.2/install-client-drivers.md

```diff
@@ -69,34 +69,22 @@ For a simple but complete "Hello World" example app, see [Build a Python App wit
 
 **Support level:** Full
 
-To install [Django](https://docs.djangoproject.com/en/3.0/topics/install/) and the [CockroachDB backend for Django](https://github.com/cockroachdb/django-cockroachdb):
+CockroachDB supports Django versions 2.2 and 3.0.
+
+To install [Django](https://docs.djangoproject.com/en/3.0/topics/install/) and the [CockroachDB backend for Django](https://github.com/cockroachdb/django-cockroachdb) for Python 3:
```

Comment:
I think you could remove "for Python 3" here. The supported versions of Django only support Python 3 (Python 2 is end of life).